src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -671,9 +673,9 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
             gmStreamFuncTopOfEdge(:, iEdge) = 0.0_RKIND
 
             ! Construct the tridiagonal matrix
-            if (maxLevelEdgeTop(iEdge) .GE. 3) then
+            if ((maxLevelEdgeTop(iEdge) - minLevelEdgeBot(iEdge)) .GE. 2) then
```

Comment:
this change seems to reproduce the original behavior, but is there a reason to switch the form instead of 
```
maxLevelEdgeTop(iEdge) .GE. minLevelEdgeBot(iEdge) + 2
```
?

Reply:
The former seemed a little more intuitive to me, but I don't mind changing it to your version.

Reply:
actually thinking about it, I agree the new change does make more intuitive sense, but we may want to make a comment that this ensures that this ensures there are at least three active cells in the column. Since we have that requirement elsewhere some may get tripped up by seeing a '2' here. But is also probably fine to leave as is.